.github/workflows/release.yml

```diff
@@ -22,15 +22,18 @@ jobs:
     ###############################
     - name: Build
       run: dotnet build
+      working-directory: "${{ env.workdir }}"
```

Comment:
I think that this should also have the same parameter and steps like in two other workflows, no?..